payjoin/Cargo.toml

```diff
@@ -22,10 +22,10 @@ bip21 = "0.3.1"
 log = { version = "0.4.14"}
 rand = { version = "0.8.4", optional = true }
 url = "2.2.2"
+bitcoind = { version = "0.31.1", features = ["0_21_2"] }
```

Comment:
PDK can't force downstream dependencies to use bitcoind, it has been generic enough to bring your own wallet and should continue to be.